source/stylesheets/refills/_phone.scss

```diff
@@ -0,0 +1,42 @@
+.phone {
+  $ratio: 1.6;
+  $inset-shadow: inset 0 1px 4px transparentize(black, 0.8);
+  $phone-padding: 0.4em * $ratio;
+  $phone-screen-width: 8.5em * $ratio;
+  $phone-screen-height: 15em * $ratio;
+  $phone-background: #F8F8F8;
+
+  background: $phone-background;
+  border-radius: (1.2em * $ratio);
+  box-shadow: 0 0 1px transparentize(black, 0.4), inset 0 3px 0 white;
+  padding: $phone-padding;
+  width: $phone-screen-width + (2 * $phone-padding);
+
+  .phone-speaker {
+    background: #ABABAB;
+    border-radius: 1em;
+    box-shadow: 0 2px 0 white, inset 0 0 1px transparentize(black, 0.6);
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.